Signum.React.Extensions/TypeHelp/TypeHelpController.cs

```diff
@@ -207,9 +209,16 @@ public TypeMemberHelpTS(Node<PropertyRoute> node, TypeHelpMode mode)
                 pr.PropertyInfo?.Name.FirstLower() :
                 pr.PropertyInfo?.Name;
 
-            this.type = mode ==  TypeHelpMode.Typescript && ReflectionServer.IsId(pr) ?
-                PrimaryKey.Type(pr.RootType).Nullify().TypeName():
-                pr.Type.TypeName();
+            this.type = mode == TypeHelpMode.Typescript && ReflectionServer.IsId(pr) ?
+                PrimaryKey.Type(pr.RootType).Nullify().TypeName() :
+                pr.Let(propertyRoute =>
+                {
+                    var typeName = propertyRoute.Type.TypeName();
+                    var nullable = propertyRoute.PropertyInfo?.GetCustomAttribute<NullableAttribute>();
+

```

Comment:
This has changed in VS 16.2, but I can fix it when I merge

Reply:
Perfect, then I can use this code till you fix and merge for VS 16.2?